dds/DCPS/QOS_XML_Handler/XML_MemBuf_Intf.h

```diff
@@ -0,0 +1,89 @@
+//==============================================================
+/**
+ *  @file  XML_MemBuf_Intf.h
+ *
+ *
+ *  @author Danilo C. Zanella (dczanella@gmail.com)
+ */
+//================================================================
+
+#ifndef OPENDDS_DCPS_QOS_XML_HANDLER_XML_MEMBUF_INTF_H
+#define OPENDDS_DCPS_QOS_XML_HANDLER_XML_MEMBUF_INTF_H
+#include /**/ "ace/pre.h"
+
+#if !defined (ACE_LACKS_PRAGMA_ONCE)
+# pragma once
+#endif /* ACE_LACKS_PRAGMA_ONCE */
+
+#include "dds_qos.hpp"
+#include "XML_Intf.h"
+#include "dds/DdsDcpsInfrastructureC.h"
+#include "OpenDDS_XML_QOS_Handler_Export.h"
+#include "ace/XML_Utils/XML_Typedefs.h"
```

Comment:
Would prefer to extend the forward declared classes below and don't inlcude XM_Utils in the header, let us now expose any internals to the user

Reply:
OK. I did it.